app/tests/spec/views/app.js

```diff
@@ -105,5 +118,316 @@ define([
         assert.isTrue(router.navigate.calledWith('signup'));
       });
     });
+
+    describe('setTitle', function () {
+      it('sets the document title', function () {
+        view.setTitle('Foo');
+        assert.equal(windowMock.document.title, 'Foo');
+      });
+    });
+
+    describe('showView', function () {
+      describe('with a view that does not render', function () {
+        var displayedView;
+        var isDestroyed = false;
+        var isLogged = false;
+
+        var DoesNotRenderView = Backbone.View.extend({
+          render: function () {
+            return p(false);
+          },
+
+          destroy: function () {
+            isDestroyed = true;
+          },
+
+          logScreen: function () {
+            isLogged = true;
+          }
+        });
+
+        before(function () {
```

Comment:
It is, although I'd have put the assignments into `before` too because I'm kind of anal like that. But also, I don't think "my" way is objectively better than any other way, maybe it is me who needs to change approach. :)